app/src/main/java/com/example/soundlesscheck_in/SpeakerFragment.java

```diff
@@ -63,15 +65,27 @@ private void setUI(View v) {
     private void getData() {
         // necessary information
         phoneNumber = EncryptedSPManager.getString(this.getActivity(), "phone");
-        livingCity = EncryptedSPManager.getString(this.getActivity(), "city");
-        String UserLoc = EncryptedSPManager.getString(requireContext(), "userCity") + " " + EncryptedSPManager.getString(requireContext(), "userTown");
+        livingCity = EncryptedSPManager.getString(requireContext(), "userCity");
+        livingTown = EncryptedSPManager.getString(requireContext(), "userTown");
+
+        positions = getCityTownInfo.stringToPosition(livingCity, livingTown).split("/");
+        mCityPosition = Integer.parseInt(positions[0]);
+        mTownPosition = Integer.parseInt(positions[1]);
+    //    Log.d("checkcitypos", mCityPosition+"/"+mTownPosition);
+    //    Log.d("realcitypos", String.valueOf(EncryptedSPManager.getInt(requireContext(), "cityPos")));
+    //    Log.d("realtownpos", String.valueOf(EncryptedSPManager.getInt(requireContext(), "townPos")));
+
+        data = phoneNumber+"/"+mCityPosition+"/"+mTownPosition;
+        // Data format : 010-xxxx-xxxx/x/x
+        // ex) 010-1234-1234/1/3 (1:city position, 3:town position)
 
-        data = phoneNumber+"/"+livingCity;
-        // Data format : 010-xxxx-xxxx/City(English)
-        // ex) 010-1234-1234/Seoul
+        // how to get city info via position number by using GetCityTownInfo class : for Listener Fragment !!
+    //    GetCityTownInfo getCityTownInfo = new GetCityTownInfo(this.getContext());
+    //    String cityInfo = getCityTownInfo.positionToString(mCityPosition, mTownPosition);
+    //    Log.d("frompos", cityInfo);
```

Comment:
You wrote an example of converting the city/county position values into strings!